build.gradle

```diff
@@ -158,9 +158,9 @@ subprojects {
                 protobuf_plugin: 'com.google.protobuf:protobuf-gradle-plugin:0.8.0',
                 protobuf_util: "com.google.protobuf:protobuf-java-util:${protobufVersion}",
 
-                netty: 'io.netty:netty-codec-http2:[4.1.4.Final]',
+                netty: 'io.netty:netty-codec-http2:[4.1.6.Final-SNAPSHOT]',
                 netty_epoll: 'io.netty:netty-transport-native-epoll:4.1.4.Final' + epoll_suffix,
-                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork19',
+                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork22',
```

Comment:
If the tests don't pass without the change, then the version bump is required. Any time we are _forced_ to bump the version, we need to inform our users, since this dependency isn't tracked by maven/gradle. Bumping the version is fine, but we just need to help out our users.